spring-cloud-skipper-server/pom.xml

```diff
@@ -45,6 +45,37 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.pivotal.spring.cloud</groupId>
+            <artifactId>spring-cloud-services-starter-config-client</artifactId>
+            <version>1.6.3.RELEASE</version>
```

Comment:
Can we define version and excludes only in one place.

Reply:
Let me try again, at the time I ended up playing wack a mole with maven.